.github/workflows/e2e-tests.yml

```diff
@@ -353,7 +352,7 @@ jobs:
         run: yarn cypress-mochawesome-to-bigquery
         working-directory: testing-tools-team-dashboard-data
         env:
-          IS_MASTER_BUILD: ${{ needs.cypress-tests-prep.outputs.is_master_build }}
+          IS_MASTER_BUILD: true
```

Comment:
```suggestion
          RUN_FULL_SUITE: true
```

Reply:
Whoops, disregard this one, didn't realize this was in the `testing-reports-cypress` job